src/controllers/order/index.ts

```diff
@@ -20,6 +38,66 @@ export class order extends Controller {
     this.db = new Database()
   }
 
+  /**
+   * Returns the details of all order demands.
+   * @summary List all order demands
+   */
+  @Get('/')
+  public async getAll(): Promise<DemandResponse[]> {
+    const capacities = await this.db.getDemands(DemandSubtype.order)
+    const result = await Promise.all(
+      capacities.map(async (order: DemandResponse) => ({
+        ...order,
+        alias: await getMemberByAddress(order.owner),
+      }))
+    )
+    return result
+  }
+
+  /**
+   * @summary Get a order by ID
+   * @param orderId The order's identifier
+   */
+  @Response<NotFound>(404, 'Item not found')
+  @Get('{orderId}')
+  public async getCapacity(@Path() orderId: UUID): Promise<DemandResponse> {
+    const [order] = await this.db.getDemand(orderId)
+    if (!order) throw new NotFound('capacity')
```

Comment:
more `capacity`s

Reply:
same as above, did not pull, did not push, please confirm